Sources/StreamChat/APIClient/APIClient_Tests.swift

```diff
@@ -450,7 +450,7 @@ extension URLSessionConfiguration {
             && urlCache == otherConfiguration.urlCache
             && shouldUseExtendedBackgroundIdleMode == otherConfiguration.shouldUseExtendedBackgroundIdleMode
 //            && protocolClasses == otherConfiguration.protocolClasses
-            && multipathServiceType == otherConfiguration.multipathServiceType
+//            && multipathServiceType == otherConfiguration.multipathServiceType
```

Comment:
This is `iOS` only property, I assume it's fine to just ignore it?

Reply:
Why not just remove it?

Reply:
I stick with already existing pattern